optimum/exporters/onnx/convert.py

```diff
@@ -884,3 +896,348 @@ def export(
     if not disable_dynamic_axes_fix:
         config.fix_dynamic_axes(output, device=device, input_shapes=input_shapes, dtype=dtype)
     return export_output
+
+
+def onnx_export_from_model(
```

Comment:
The function is also removed to `onnx_export_from_model` to explicit its role, and documented.